test/qunit/model/FHIRModel.unit.js

```diff
@@ -215,9 +215,11 @@ sap.ui.define([
 	});
 
 	QUnit.test("check listbinding getcontexts in relative case when context is set but no data is in the model", function(assert) {
+		this.oFhirModel1.setProperty(this.sPatientPath + "/resourceType", "Patient");
 		var aContexts = this.oListBinding3.getContexts();
 		assert.deepEqual(aContexts, [], "The contexts matches");
 		assert.strictEqual(this.oListBinding3.getLength(), 0, "The total length matches");
+		this.oFhirModel1.setProperty(this.sPatientPath, undefined);
```

Comment:
why we set the value to undefined here? There no further asserts. I don't get the goal of this.

Reply:
removed.